pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -218,20 +246,43 @@ func (r *AWSLoadBalancerControllerReconciler) SetupWithManager(mgr ctrl.Manager)
 		Owns(&appsv1.Deployment{}).
 		Owns(&corev1.Service{}).
 		Owns(&arv1.ValidatingWebhookConfiguration{}).
-		Owns(&arv1.MutatingWebhookConfiguration{}).
-		Complete(r)
+		Owns(&arv1.MutatingWebhookConfiguration{})
```

Comment:
You removed the call to `Complete` here.  What does the work that `Complete` used to do (actual building)?